src/Components/SysTable/Helpers.test.js

```diff
@@ -0,0 +1,80 @@
+import { buildApiFilters } from './Helpers';
+
+describe('buildApiFilters', function () {
```

Comment:
NItpick: I feel a test to cover otherFilters involvement could be written just to have 101% coverage

Reply:
Is this what you had in mind? https://github.com/RedHatInsights/malware-detection-frontend/pull/117/files#diff-ade7153372810d51e2355c4e393b1d40341ada49e413ed97868c063be437258eR8-R11